CHANGELOG

```diff
@@ -15,38 +15,32 @@ Starting with this version, the MFEM open source license is changed to BSD-3.
 
 Improved GPU support
 --------------------
-- Added support for matrix-free diagonal smoothers on GPUs.
-
-- Added a "debug" device that is to be used specifically for debugging purposes.
-  It allows to test the "device" code-path (using separate host/device memory
-  pools and host <-> device transfers) without any GPU hardware.
-
 - Added initial support for AMD GPUs based on HIP: a C++ runtime API and kernel
-  language that can run on both AMD and NVIDIA hardware. With this change and
-  the libCEED addition below, the current list of available backends is:
-  "ceed-cuda", "occa-cuda", "raja-cuda", "cuda", "hip", "debug", "occa-omp",
-  "raja-omp", "omp", "ceed-cpu", "occa-cpu", "raja-cpu", and "cpu".
+  language that can run on both AMD and NVIDIA hardware. With this change plus
+  the libCEED and debug devise additions below, the current list of available
+  backends is: "ceed-cuda", "occa-cuda", "raja-cuda", "cuda", "hip", "debug",
+  "occa-omp", "raja-omp", "omp", "ceed-cpu", "occa-cpu", "raja-cpu", and "cpu".
 
-- Improved RAJA backend and multi-GPU MPI communications.
-
-Memory Manager Backends
------------------------
 - Added support for Umpire, a resource management library that allows the
   discovery, provision, and management of memory on machines with multiple
-  memory devices like NUMA and GPUs, https://github.com/LLNL/Umpire.
-
-- The memory manager now supports different new memory types, associated
-  with the following backends:
-    - Standard C++, using 'std::new' and 'std::delete',
-    - Aligned 32 or 64 bits, using '_aligned_malloc' on WIN32 architectures,
-      'posix_memalign' otherwise,
-    - CUDA pointers, using 'cudaMalloc',
-    - HIP pointers, using 'hipMalloc',
-    - Managed CUDA/HIP pointers (UVM), using 'cudaMallocManaged'/
-      'hipMallocManaged', respectively,
-    - Debug memory types with protection, emulating separate host and device
-      memory pools, using 'mmap' and 'mprotect'; used by the new "debug" device,
-    - Umpire, using its API.
+  memory devices like NUMA and GPUs, see https://github.com/LLNL/Umpire.
+
+- Added a "debug" device designed specifically to aid in debugging GPU code by
+  following the "device" code path (using separate host/device memory spaces and
+  host <-> device transfers) without any GPU hardware.
+
+- The MFEM memory manager now supports different memory types, associated with
+  the following memory backends:
+  * Standard host memory, using C++ std::new and std::delete,
```

Comment:
I just realized that, strictly speaking, `std::new` and `std::delete` do not make sense since they are both keywords in C++. It may be good to paraphrase this bullet.

Reply:
Does 7f5a8e9 look good? I'm ready to merge this...